tests/test_api_build.py

```diff
@@ -806,10 +771,7 @@ def test_legacy_noarch_python(testing_config):
     assert os.path.basename(os.path.dirname(output)) == 'noarch'
 
 
-@pytest.mark.skipif(True,
-                    reason="Re-enable when private application environments are fully implemented "
-                           "in conda. "
-                           "See https://github.com/conda/conda/issues/3912#issuecomment-374820599")
+@pytest.mark.skip(reason="The Test is using old conda versions that will be deprecated in the next prunning process.. Skipping this test case till then")
 def test_preferred_env(testing_config):
```

Comment:
Remove this test. It's for a feature similar to the conda private env (https://github.com/conda/conda/pull/12101) that was never fully implemented in conda.